src/vaults/src/handlers/bribe-processor-handler.ts

```diff
@@ -0,0 +1,15 @@
+import { TreeDistribution } from '../../generated/BribeProcessor/VotiumBribeProcessor';
+import { BadgerTreeDistribution } from '../../generated/schema';
+import { BVECVX } from '../constants';
+
+export function handleTreeDistribution(event: TreeDistribution): void {
```

Comment:
https://github.com/Badger-Finance/badger-subgraph/blob/e375b7fc37486ca6b4cc75178db52b79e2f8d667/src/vaults/src/handlers/strategy-handler.ts#L78-L85

we could use this one instead